### PR TITLE
[Outreachy Task Submission] Background and foreground colors do not have a sufficient contrast ratio on Post card 

### PR DESCRIPTION
**Pull request by JUNNY12**
### Description
This PR addresses the accessibility issue I identified in [Issue #4842](https://github.com/ushahidi/platform/issues/4842), where the background and foreground colors on the Post card did not have a sufficient contrast ratio. The lack of contrast made it difficult for some users, particularly those with visual impairments, to read the content on the card.

### Solution
To resolve this issue, this PR adjusts the colors to ensure that the contrast ratio between the background and foreground meets the Web Content Accessibility Guidelines (WCAG) standards. By improving the color contrast, this change enhances the readability and usability of the Post card for all users.

### Screen record after fix

https://github.com/ushahidi/platform-client-mzima/assets/81489510/f6d6b45f-b84f-4385-99bd-af4a318e982d

### Purpose
This PR is a part of my submission for the Outreachy task, and it contributes to improving the accessibility of the Ushahidi platform.

### CODE REVIEW

**Comment by JUNNY12** [comment]
@Angamanga Please Kindly Review this

File: apps/web-mzima-client/src/env.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "production": true,
-  "backend_url": "http://localhost:8080/",
+  "backend_url": "https://outreachyushahidi.api.ushahidi.io/",
   "api_v3": "api/v3/",
```
Comment (Angamanga) [inline apps/web-mzima-client/src/env.json]: Keep the localhost:8080 url here

**Review by Angamanga — CHANGES_REQUESTED** [review]
Thank you for your contribution! A small change is needed before approving 🙏

File: apps/web-mzima-client/src/env.json (hunk 1 above)
Comment (JUNNY12) [reply apps/web-mzima-client/src/env.json]: > Keep the localhost:8080 url here

Thank you @Angamanga for the review. I have make the necessary changes.

Comment (JUNNY12) [reply apps/web-mzima-client/src/env.json]: Good day @Angamanga, This is a reminder to kindly review this changes

**Review by Angamanga — APPROVED** [review]
Thank you! Approved 🎉

**Comment by JUNNY12** [comment]
> Thank you! Approved 🎉

Thank you @Angamanga . Its my honor to be part of this community

**Pull request closed without merging**